MonoGame.Framework/Matrix.cs

```diff
@@ -7,13 +7,35 @@
 using System.Runtime.Serialization;
 
 namespace Microsoft.Xna.Framework
-{
+{ 
+    /// <summary>
+    /// Represents the right-handed 4x4 float-based matrix, which can store translation,scale and rotation information.

```

Comment:
`float-based` is weird.... just say `floating point`.
Put a space between `translation,scale`.